pkgs/applications/video/mpv/wrapper.nix

```diff
@@ -1,14 +1,84 @@
-{ stdenv, symlinkJoin, makeWrapper, mpv, scripts ? [] }:
+# Arguments that this derivation gets when it is created with `callPackage`
+{ stdenv
+, lib
+, symlinkJoin
+, makeWrapper
+, mpv
```

Comment:
This argument is not needed, as the required argument Line 11/10 is used eventually.

Reply:
Fixed in last push.